hapi/hedera-protobufs/services/token_claim_airdrop.proto

```diff
@@ -48,17 +49,17 @@ import "basic_types.proto";
  * SHALL NOT be restated in the record file.<br/>
  * All claims MUST succeed for this transaction to succeed.
  *
- * ### Record Stream Effects
+ * ### Block Stream Effects
```

Comment:
This brings up an interesting point: do we not want associated record stream effects specified in these files too, until we make the switch to block streams?

Reply:
Probably not worthwhile, also we'd be publicly documenting something we're removing in the not-distant future...